src/openloco/graphics/gfx.cpp

```diff
@@ -387,15 +395,19 @@ namespace openloco::gfx
                     break;
 
                 default:
-                    if (chr >= 32)
-                    {
-                        gfx::draw_image_palette_set(context, pos.x, pos.y, 1116 + chr - 32 + _currentFontSpriteBase, _textColours);
-                        pos.x += _characterWidths[chr - 32 + _currentFontSpriteBase];
-                    }
-                    else
+                    if (!offscreen)
                     {
-                        // Unhandled control code
-                        assert(false);
+                        // We are offscreen in the y dimension. No requirement to keep pos.x correct
```

Comment:
Given the `!offscreen` condition above, is this comment correct?

Reply:
Yup thats wrong it should read `When we are offscreen in the y dimension there is no requirement to keep pos.x correct`